src/shared/components/plots/PlotsTab.tsx

```diff
@@ -2476,6 +2585,10 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
                 dataTypeIds.push(CLIN_ATTR_DATA_TYPE);
             }
 
+            if (this.customAttributeOptions.result!.length) {
```

Comment:
This looks not very clear, maybe we can replace with `_.isEmpty` from lodash https://lodash.com/docs/4.17.15#isEmpty
```suggestion
            if (!_.isEmpty(this.customAttributeOptions.result)) {
```

Reply:
Done